src/binary/ibc-relayer/commands/start.ts

```diff
@@ -29,6 +34,14 @@ type Options = {
 };
 
 export async function start(flags: Flags) {
+  const logLevel = resolveOption(
+    flags.logLevel,
+    process.env.RELAYER_LOG_LEVEL,
+    'info'
+  );
+
+  const logger = createLogger(logLevel as Level);
```

Comment:
I would do some validation here not naively casting string to a set of valid strings.
But yeah, for the next PR